server/lib/flow-event.js

```diff
@@ -44,26 +44,63 @@ const UTM_PATTERN = /^[\w.%-]+$/;
 
 const IS_DISABLED = config.get('client_metrics').stderr_collector_disabled;
 
+const PERFORMANCE_TIMINGS = [
+  // These timings are only an approximation, to be used as extra signals
+  // when looking for correlations in the flow data. They're not perfect
+  // representations, for instance:
+  //
+  //   * `network` includes fetching from the browser cache.
+  //   * `server` includes some network time.
+  //   * `client` is only a subset of the client-side processing time.
+  //
+  // Bear this in mind when looking at the data. The main `flow.performance`
+  // event represents our best approximation of overall, user-perceived
+  // performance.
```

Comment:
I'll also add appropriate comments to the flow event docs in the auth server, if/when this gets merged.